src/ecuacion/MayorMenorNotas.java

```diff
@@ -0,0 +1,181 @@
+package ecuacion;
+import java.util.Scanner;
+import java.time.LocalDate;
+public class MayorMenorNotas {
+
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in);
+		boolean program = false;
+		int menosNotas= Integer.MAX_VALUE;
+		int masNotas = Integer.MIN_VALUE;
+		int menorPromedio = Integer.MAX_VALUE;
+		int mayorPromedio = Integer.MIN_VALUE;
+		String menosCantidadNotas="";
+		String masCantidadNotas="";
+		String mejorPromedio="";
+		String peorPromedio="";
+		int aux = 0;
+		int mayorEdad=0;
+		int menorEdad=0;
+		int masculino=0;
+		int femenino=0;
+		int programacion = 0;
+		int fisica=0;
+		int matematica=0;
+		int materia=0;
+		do {	
+			System.out.println("Cuantos estudiantes desea ingresar?");
+			int numStudent = scan.nextInt();
+			for(int i=0; i<numStudent;i++) {
+				System.out.println("Nombre del estudiante numero "+ (aux+1));
+				scan.nextLine();
+				String nameStudent = scan.nextLine();
+				//Fecha de nacimiendo del estudiante
+				System.out.println("Ingrese la fecha de nacimiento del estudiante del estudiante: ");
+				System.out.print("Ingrese el año: ");
+				int yearBorn= scan.nextInt();
+				System.out.print("Ingrese el mes: ");
+				int monthBorn= scan.nextInt();
+				System.out.print("Ingrese el dia: ");
+				int dayBorn= scan.nextInt();
+				
+				//Sexo del estudiante
+				System.out.println("Ingrese el tipo sexo del estudiante H/Hombre  M/Mujer");
+				String sex = scan.next();
+				switch(sex) {
+					case "H":
+						masculino++;
+					break;
+					
+					case "h":
+						masculino++;
+					break;
+					
+					case "M":
+						femenino++;
+					break;
+					
+					case "m":
+						femenino++;
+					break;
+					
+					default:
+						System.out.println("No binario");
+				}
+				
+				
+				//fecha actual en años, meses y dias
+				LocalDate today = LocalDate.now();
+				int yearDate = today.getYear();
+				int monthDate = today.getMonthValue();
+				int dayDate = today.getDayOfMonth();
+				int edad = 0;
+				
+				//Calculo de la edad
+				if(yearDate > yearBorn) {
+					edad = yearDate-yearBorn;
+					if(monthDate < monthBorn) {
+						edad--;
+					}else if(monthDate == monthBorn) {
+						if(dayDate < dayBorn) {
+							edad--;
+						}
+					}
+				}		 
+				int edadActual = edad;
+				if(edadActual >= 18) {
+					System.out.println("El estudiante es mayor de edad, tiene "+ edadActual);
+					 mayorEdad++;
+					
+				}else {
+					System.out.println("El estudiante es menor de edad, tiene "+ edadActual);
+					menorEdad--;
+				}
+				//numero de notas
+				System.out.println("Cantidad de notas del estudiante ");
+				int numRating = scan.nextInt();
```

Comment:
Spanglish, don't use this, or English or Spanish, not both.

Reply:
JAHAJAAHAHAHAHAJAJAJA